corehq/apps/app_manager/management/commands/fix_title_label.py

```diff
@@ -20,11 +19,13 @@ def handle(self, **options):
                 for module in app.modules:
                     if hasattr(module, 'search_config'):
                         print(f"Looking at Module: {module.name[app.default_language]}")
-                        search_config = getattr(module, "search_config")
-                        title_label = getattr(search_config, 'title_label')
-                        new_labels = title_label.copy()
-                        for lang, string in title_label.items():
-                            if string is None:
-                                print(f"Fixing translation: {{{lang}: {string}}}")
-                                new_labels[lang] = ""
-                        setattr(search_config, 'title_label', new_labels)
+                        try:
+                            title_label = module.search_config.title_label
+                            for lang, string in title_label.items():
+                                if string is None:
+                                    print(f"Fixing translation: {{{lang}: {string}}}")
+                                    module.search_config.title_label[lang] = ""
+                                    app.save()
+                        except:
+                            print("Skipping Module.")
```

Comment:
In case there's a module without a `title_label` attribute. There may be some apps that were created while the `title_label` code was not deployed so this was meant to handle it.

Reply:
Okay so it sounds like I can just take out the `try except` lines and just keep the rest. Got it.